exercises/gigasecond/src/lib.rs

```diff
@@ -1,7 +1,10 @@
-extern crate chrono;
-use chrono::*;

```

Comment:
Agree that we should remove the glob import as bad style, but I think we can do better. What about:

```rust
use chrono::{DateTime, Duration};
```

This way we don't have to mess with a second dependency or globs.

Reply:
This applies both to the lib stub and the example.